### PR TITLE
add jspsych-spatial-orientation.js plugin

### PR DESCRIPTION
**Pull request by Floozutter**
I also started `spatial-orientation.js` (the task) to show the plugin in action.

### CODE REVIEW

File: baseline/client/js/jspsych-spatial-orientation.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            targetAngle: trial.targetAngle,
+            onClick: clickData => {
+                // build data
+                const rt = performance.now() - start;
+                const data = {
+                    ...clickData,
+                    rt: rt,
+                };
+                // finish trial
+                setTimeout(() => { jsPsych.finishTrial(data); }, 1000);
```
Comment (noah10) [inline baseline/client/js/jspsych-spatial-orientation.js:136]: I didn't see anything in the specs about time limits on individual trials (just on the experiment overall). Did I just miss it, or is this some temporary testing feature? It looks like trials are supposed to end when the user presses the enter key, which I'm assuming you haven't gotten to yet.

File: baseline/client/js/jspsych-spatial-orientation.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        });
+        draw(0);
+    };
+
+    plugin.trial = (displayElement, trial) => {
+        // build and show display HTML
+        {
+            let html = "";
+            html += `<div id="jspsych-spatial-orientation-wrapper">`;
+            html +=     `<div id="jspsych-spatial-orientation-scene">${trial.scene}</canvas>`;
```
Comment (noah10) [inline baseline/client/js/jspsych-spatial-orientation.js:113]: I think you want `</div>` here, not `</canvas>`.

File: baseline/client/js/jspsych-spatial-orientation.js (hunk 1 above)
Comment (noah10) [reply baseline/client/js/jspsych-spatial-orientation.js:136]: Now that I've played with it I see that you're finishing the trial on click, not on enter. I like that better, so as long as it's ok with Hyun Joo that's great. Is there any reason to wait a second after the click to finish it, though?

**Review by noah10 — CHANGES_REQUESTED** [review]
What's here looks good. If you can fix the misplaced </canvas> tag I'll go ahead and merge all of this in.

File: baseline/client/js/jspsych-spatial-orientation.js (hunk 1 above)
Comment (Floozutter) [reply baseline/client/js/jspsych-spatial-orientation.js:136]: The delay is there to show the `targetAngle` red line!

File: baseline/client/js/jspsych-spatial-orientation.js (hunk 2 above)
Comment (Floozutter) [reply baseline/client/js/jspsych-spatial-orientation.js:113]: Ah, that's right.

**Review by noah10 — APPROVED** [review]

**Pull request merged**